docs/maintainers/generic-how-to-implement-a-concept-exercise.md

```diff
@@ -19,7 +19,12 @@ languages
                 |   ├── instructions.md
                 |   ├── introduction.md
                 |   ├── hints.md
+<<<<<<< HEAD
```

Comment:
@SaschaMann I think something went wrong with the merging. This also applies to the other files.

Reply:
Good catch. For some reason VS Code didn't display those to me, or I missed them.